modules/account-lib/test/unit/coin/near/transaction.ts

```diff
@@ -32,13 +32,25 @@ describe('Near Transaction', () => {
   });
 
   describe('from raw transaction', () => {
-    it('build a transfer from raw', async () => {
+    it('build a transfer from raw hex', async () => {
```

Comment:
Just a comment:
@alebusse why you are no covering signed tx?

Reply:
Its just the title, thats actually the signed one. I will fix it